### PR TITLE
ctl: add a message if the scheduler does not exist

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Ref #4399

### What is changed and how does it work?

before 
```bash
./pd-ctl scheduler config evict-leader-scheduler  add-store 2
Failed! [404] 404 page not found
```
After
```bash
./pd-ctl scheduler config evict-leader-scheduler  add-store  2
Failed! scheduler evict-leader-scheduler not found.
```

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
ctl: add a message if the scheduler does not exist
```

### CODE REVIEW

File: tools/pd-ctl/tests/scheduler/scheduler_test.go
Comment (lhy1024) [inline tools/pd-ctl/tests/scheduler/scheduler_test.go:324]: Do other schedulers need it? For example grant-leader-scheduler.

Comment (HuSharp) [reply tools/pd-ctl/tests/scheduler/scheduler_test.go:324]: yes, I added in `grant-leader-scheduler` and `evict-leader-scheduler`

**Review by lhy1024 — APPROVED** [review]

**Comment by HuSharp** [comment]
@okJiang PTAL, thx!

File: tools/pd-ctl/pdctl/command/scheduler.go
Comment (okJiang) [inline tools/pd-ctl/pdctl/command/scheduler.go:650]: It is better to discuss this with PM.

```suggestion
		cmd.Printf("Can't update config for a non-existent scheduler.\n", schedulerName)
```

File: tools/pd-ctl/pdctl/command/scheduler.go
Comment (okJiang) [inline tools/pd-ctl/pdctl/command/scheduler.go:786]: ditto

**Review by okJiang — COMMENTED** [review]
Please add an issue for this pr

**Review by rleungx — APPROVED** [review]

**Comment by rleungx** [comment]
Let's wait for @okJiang to give an approval.

**Comment by HuSharp** [comment]
After discussing with PM. He gives suggestions for these 2 issues:
> ask: what will error msg be like?

PM: The failed msg is `Unable to update config: schedulerXX does not exist.`

> ask: Also, I noticed there are 2 ways for evictLeader and grantLeader:
1. `./pd-ctl scheduler add evict-leader-scheduler <store-id>`
If there is no evict leader scheduler before, no error will be reported and the scheduler for this store will be generated
2. `./pd-ctl scheduler config evict-leader-scheduler add-store <store_id>`
If there is no evict leader scheduler before, it will report the error: `Failed! [404] 404 page not found`

PM: Since add and config are two subcommands, it doesn't seem necessary to unify them, which makes sense.

cc @songrijie @okJiang @rleungx

File: tools/pd-ctl/pdctl/command/scheduler.go
Comment (HuSharp) [reply tools/pd-ctl/pdctl/command/scheduler.go]: changed to `Unable to update config: schedulerXX does not exist.` PTAL, thx~

**Review by okJiang — APPROVED** [review]

**Pull request merged**
